ground/src/main/java/com/google/android/ground/ui/map/gms/renderer/PolygonRenderer.kt

```diff
@@ -60,9 +61,10 @@ class PolygonRenderer(
     polygon.holes.forEach { options.addHole(it.coordinates.toLatLngList()) }
 
     val mapsPolygon = map.addPolygon(options)
+    val strokeScale = if (isSelected) 2f else 1f
```

Comment:
We should define thisin a resource file or constant, but we can come back to this later.